TUM-Dev/NavigaTUM
# [Cal-13] Calendar Frontend

Fixes #289 

Depends on #637 being merged

## Proposed Changes

- Implemented a calendar modal
  ![image](https://user-images.githubusercontent.com/26258709/227743121-ccdec2dc-403d-41b6-9413-5ce73a3711c0.png)

## How to test this PR

1. Launch this PR
2. See if this matches the tumonline link

## How has this been tested?

- [x] manual validation via select examples

## Checklist:
